staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -462,6 +458,37 @@ func (cfgCtlr *configController) digestConfigObjects(newPLs []*flowcontrol.Prior
 	return suggestedDelay, utilerrors.NewAggregate(errs)
 }
 
+func updateOrApply(client flowcontrolclient.FlowSchemaInterface, fsu fsStatusUpdate, asFieldManager string) error {
+	if utilfeature.DefaultFeatureGate.Enabled(features.ServerSideApply) {
```

Comment:
@liggitt - we're generally locking GA features to true and SSA is already GA, but not locked:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L234

What was the reason for it?

[I'm asking because if it was locked, then we wouldn't need this conditional logic here.]

Reply:
cc @apelisse - I'd be in favor of locking/dropping the conditional apply gate logic

Reply:
Opened https://github.com/kubernetes/kubernetes/pull/112748

Reply:
@tkashem - this is now merged
Once the dependent PR are merged you can update this PR and get rid of this condition - this if will now always be true.